### DISCUSSION TITLE
2005/09/29/%E5%A6%82%E4%BD%95%E6%9F%A5%E6%89%BE%E8%8B%B1%E6%96%87%E4%B8%93%E4%B8%9A%E8%AF%8D%E6%B1%87%E6%89%80%E5%AF%B9%E5%BA%94%E7%9A%84%E4%B8%AD%E6%96%87%E7%BF%BB%E8%AF%91

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#51] [Announcements]
# 2005/09/29/%E5%A6%82%E4%BD%95%E6%9F%A5%E6%89%BE%E8%8B%B1%E6%96%87%E4%B8%93%E4%B8%9A%E8%AF%8D%E6%B1%87%E6%89%80%E5%AF%B9%E5%BA%94%E7%9A%84%E4%B8%AD%E6%96%87%E7%BF%BB%E8%AF%91

这是我的办法：一般的词汇，如果大家最常用什么中文翻译，我就也用什么翻译。我是在 google 上查找翻译的。

https://blog.qizhen.xyz/2005/09/29/%E5%A6%82%E4%BD%95%E6%9F%A5%E6%89%BE%E8%8B%B1%E6%96%87%E4%B8%93%E4%B8%9A%E8%AF%8D%E6%B1%87%E6%89%80%E5%AF%B9%E5%BA%94%E7%9A%84%E4%B8%AD%E6%96%87%E7%BF%BB%E8%AF%91

**Comment by ruanqizhen** [comment]
锋
2010年05月1日 11:38 编辑
一般用dict.cnki.net，但是不太规范，都是国人写的。

回复
林
2007年02月26日 15:54 编辑
用爱词霸，非常好

回复
Clara
2006年07月27日 12:02 编辑
 I never know that Goole can be used like that. ——–English teacher here, ang got your blog address from others. Thank you so much.:)

回复
流连
2005年09月29日 20:35 编辑
看了你的荨麻疹经历。嘿嘿~~跟我很像阿。可是我现在不去看病也不吃药了。就是当它不存在，不治了。祝你早日康复！谢谢你关注我很久以前的blog，想问一下，你怎么有耐心翻看以前的文章呢？真巧…….

回复
